Silex/EkinoNewRelicServiceProvider.php

```diff
@@ -1,117 +0,0 @@
-<?php
```

Comment:
Here you are removing the support of (the deprecated framework) silex wich is a BC break for this lib and should be done IMHO in a dedicated PR